src/main/java/com/team4/testingsystem/services/impl/FileAnswerServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.team4.testingsystem.services.impl;
+
+import com.team4.testingsystem.dto.FileAnswerRequest;
+import com.team4.testingsystem.entities.FileAnswer;
+import com.team4.testingsystem.exceptions.NotFoundException;
+import com.team4.testingsystem.repositories.FileAnswerRepository;
+import com.team4.testingsystem.repositories.QuestionRepository;
+import com.team4.testingsystem.services.FileAnswerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FileAnswerServiceImpl implements FileAnswerService {
+    private final FileAnswerRepository fileAnswerRepository;
+    private final QuestionRepository questionRepository;
+
+    @Autowired
+    public FileAnswerServiceImpl(FileAnswerRepository fileAnswerRepository, QuestionRepository questionRepository) {
+        this.fileAnswerRepository = fileAnswerRepository;
+        this.questionRepository = questionRepository;
+    }
+
+    public void create(FileAnswerRequest fileAnswerRequest) {
+        FileAnswer fileAnswer = FileAnswer.builder()
+                .url(fileAnswerRequest.getUrl())
+                .question(questionRepository.findById(fileAnswerRequest.getQuestionId())
+                        .orElseThrow(NotFoundException::new))
+                .build();
+        fileAnswerRepository.save(fileAnswer);
+    }
+
+    public FileAnswer getById(long id) {
+        return fileAnswerRepository.findById(id)
+                .orElseThrow(NotFoundException::new);
```

Comment:
Maybe we should add `FileAnswerNotFoundException extends NotFoundException`, since we have exceptions for levels, modules, etc.

Reply:
There is already `FileNotFoundException` in develop, so we can use it here. We can also create `FileAnswerNotFoundException` and rename  `FileNotFoundException` to `ContentFileNotFoundException`